src/app/services/app-state.service.ts

```diff
@@ -75,8 +110,20 @@ export class AppStateService {
   }
 
 
+  /**
+   * Stores the selected record in the
+   *
+   * @param rowId
+   * @param doNotReloadForm
+   */
   setRecordSelected(rowId: number | null, doNotReloadForm: boolean = false): void {
-    this.onRecordSelected$.next({ cinchyId: rowId, doNotReloadForm });
+
+    this._rowId = rowId;
```

Comment:
This is the most important line in the PR. This assignment is what actually sets the state in this service.